mainnet_v1/gw-mainnet_v1-config-readonly.toml

```diff
@@ -180,80 +58,18 @@ max_cycles_limit = '1950000000'
-
-[consensus.contract_type_scripts.custodian_lock]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0x3ef49f39b45bd41a5bcc8ef103e88c675e850c1de2b51b1cb3c70edd6ee56936'
-
-[consensus.contract_type_scripts.withdrawal_lock]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0xffc2139a9fb7d05fd5927d537eb8ed692cb67d29404fc710333f01dbfd17043a'
-
-[consensus.contract_type_scripts.challenge_lock]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0xb0cf1d97b353716058a5ce70fa6bd2d4014ed5a14e8307038c56cad726cd457e'
-
-[consensus.contract_type_scripts.l1_sudt]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0xcf37df53ec0eb890937879ad5587521f737dc3d95eb1c22194b958ecbcf39a8b'
-
-[consensus.contract_type_scripts.omni_lock]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0xf3a8efcb165546ab0d7bc2aaf35c52c3432c571b4b1b2abee0619e872539654d'
-[consensus.contract_type_scripts.allowed_eoa_scripts.0x096df264f38fff07f3acd318995abc2c71ae0e504036fe32bc38d5b6037364d4]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0x9fa95a4ca1d71ea942940eca96ca8e8f10904322daa00461755724cf065075b2'
-[consensus.contract_type_scripts.allowed_contract_scripts.0x83d5d8841518e8db686909d27c821398491f475ed5f1cd392c36e83f4252c4ac]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0xaf85ca80c12364dc26e04f7e823219cfe284b3f47ae95771ea10d16bdb704042'
-
-[consensus.contract_type_scripts.allowed_contract_scripts.0x2c0b586137cb942f92cc3f84f62d77924b560e4088667f6f47012ecfa828e184]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0x9b913ed521d53aeebe2e6a66b142ce6313b32d84a53b76c4e9df53f52c5acaf7'
-
-[consensus.contract_type_scripts.allowed_contract_scripts.0xc55c5ede907d13ac1e744cff8ce4386b9dc5aad905d8f3badfd9419efe49eeb2]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0xc7e05ffd9b1447d4f96b7c03b24ba0a24c57dc620e1218b69216e0fda3d62d3d'
-
-[consensus.contract_type_scripts.allowed_contract_scripts.0x990027acd7058ec1b45df9d7448c0c5407fc17dde4b9b769f594d613c8053084]
-code_hash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
-hash_type = 'type'
-args = '0x22ee82c1c021a8371f1475e425610fb4939b452bdee648e930cb1c8bcf48640d'
-[dynamic_config.fee_config]
+[mem_pool.fee]
 meta_cycles_limit = 20000
 sudt_cycles_limit = 20000
 eth_addr_reg_cycles_limit = 20000
 withdraw_cycles_limit = 20000
 
-[dynamic_config.rpc_config]
+[mem_pool.extra]
 allowed_sudt_proxy_creator_account_id = [3]
-sudt_proxy_code_hashes = []
+sudt_proxy_code_hashes = [] 
```

Comment:
```suggestion
sudt_proxy_code_hashes = ["0xde4542f5a5bd32c09cd98e9752281f88900a059aab7ac103edd9df214f136c52"]

```

Reply:
I see the readonly nodes ```sudt_proxy_code_hashes```  is empty

Reply:
yes, it was ok.
`sudt_proxy_code_hashes` is an optional config on a gw-readonly-node, while it is an important config on gw-fullnode.

Let's make this config clear in a gw-readonly-node too.

## Related Code
### Only accounts in allow list could create sUDT proxy contract.
https://github.com/godwokenrises/godwoken/blob/118163d10fd62761463af8c788f32aa23c98e365/crates/tx-filter/src/erc20_creator_allowlist.rs#L22-L50

Reply:
OK